### PR TITLE
Completed javascript-timers Exercise - Bingle Padilla

### PR DESCRIPTION
**Pull request by codewithbingle**
javascript-timers-timeout
javascript-timers-interval

### CODE REVIEW

File: javascript-timers/interval.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+var $countdownDisplay = document.querySelector('.countdown-display');
+var $startingNumber = parseInt($countdownDisplay.textContent);
+
+function countdownDisplay() {
+  if ($startingNumber > 1) {
+    $startingNumber--;
+    $countdownDisplay.textcontent = $startingNumber;
+  } else {
+    $countdownDisplay.textContent = '~Earth Beeeelooowww Us~';
+    clearInterval(start);
```
Comment (TimelikeClosure) [inline javascript-timers/interval.js]: What value does `start` have? `clearInterval` expects the id of the interval to clear.

**Review by TimelikeClosure — CHANGES_REQUESTED** [review]
---

File: javascript-timers/interval.js (hunk 1 above)
Comment (codewithbingle) [reply javascript-timers/interval.js]: clearInterval should be the intervId.

Comment (codewithbingle) [reply javascript-timers/interval.js]: Updated!

**Review by TimelikeClosure — APPROVED** [review]

**Pull request merged**